### PR TITLE
Fix demo deploy after introducing integrated TURN servers

### PR DESCRIPTION
**Pull request by FelonEkonom**

### CODE REVIEW

File: webrtc/videoroom_advanced/config/runtime.exs [hunk 1]
```diff
@@ -71,6 +71,21 @@ defmodule ConfigParser do
       {:error, :einval} -> addr
     end
   end
+
+  def parse_integrated_turn_port_range(range) do
+    with [str1, str2] <- String.split(range, "-"),
+         from when from >= 0 and from <= 65_535 <- String.to_integer(str1),
+         to when to >= 0 and to <= 65_535 and from <= to <- String.to_integer(str2) do
```
[the remark this answers is not in the document]
Comment (FelonEkonom) [reply webrtc/videoroom_advanced/config/runtime.exs:78]: I think, that suggested change might be not optimalised and lead to checking, if specific number is element of set of 65 535 elements

[the remark this answers is not in the document]
Comment (mat-hek) [reply webrtc/videoroom_advanced/config/runtime.exs:78]: That's not true, ranges are optimized for this and this is the idiomatic way of doing such comparisons in elixir

File: .github/workflows/webrtc_videoroom_advanced_sandbox_build_and_deploy.yml [hunk 2]
```diff
@@ -53,8 +53,8 @@ jobs:
       
       - name: Change Docker-Compose address before deployment
         id: sed_it_again
-        run: sed -i -e "s/demo-/sandbox-/g" webrtc/videoroom_advanced/docker-compose.yml
-
+        run: sed -i -e "6,9 s/videoroom/sandbox-videoroom/" webrtc/videoroom_advanced/docker-compose.yml
+
```
Comment (mickel8) [inline .github/workflows/webrtc_videoroom_advanced_sandbox_build_and_deploy.yml:57]: What's that?

File: webrtc/videoroom_advanced/docker-compose.yml [hunk 3]
```diff
@@ -5,7 +5,12 @@ services:
     environment:
       VIRTUAL_HOST: "videoroom.membraneframework.org"
       LETSENCRYPT_HOST: "videoroom.membraneframework.org"
-      USE_INTEGRATED_TURN: "false"
+      USE_INTEGRATED_TURN: "true"
+      INTEGRATED_TURN_IP: "135.181.203.255"
+      INTEGRATED_TURN_PORT_RANGE: "50000-59999"
```
Comment (mickel8) [inline webrtc/videoroom_advanced/docker-compose.yml:10]: Shouldn't be here also something like `INTEGRATED_TURN_MOCK_IP`

**Review by mickel8 — CHANGES_REQUESTED** [review]

File: .github/workflows/webrtc_videoroom_advanced_sandbox_build_and_deploy.yml (hunk 2 above)
Comment (FelonEkonom) [reply .github/workflows/webrtc_videoroom_advanced_sandbox_build_and_deploy.yml:57]: This needed to use this same values of `VIRTUAL_HOST` and `LETSENCRYPT_HOST` in `docker-compose.yml` on demo and on sandbox

**Review by mickel8 — APPROVED** [review]

**Pull request merged**